src/attorney.ts

```diff
@@ -10,7 +10,7 @@ const POLICY = {
 
 function validateQueueArgs (config: any = {}) {
   assert(!('deadLetter' in config) || config.deadLetter === null || (typeof config.deadLetter === 'string'), 'deadLetter must be a string')
-  assert(!('deadLetter' in config) || config.deadLetter === null || /[\w-]/.test(config.deadLetter), 'deadLetter can only contain alphanumeric characters, underscores, or hyphens')
+  assert(!('deadLetter' in config) || config.deadLetter === null || /^[\w-]+$/.test(config.deadLetter), 'deadLetter can only contain alphanumeric characters, underscores, or hyphens')
```

Comment:
As I understand this, non-usage of simple `.` periods is enforced. Sadly, I actually have many queues with them being present. Is this really necessary? If so: how do I migrate away?

Reply:
also: wouldn't it make sense to define this regex in one place instead of 3?